schemas/system/domains.schema

```diff
@@ -0,0 +1,8 @@
+{
+  IdStatus: Enum('Preallocated', 'Initialization', 'Actual', 'Historical'),
+  StorageKind: Enum('Master', 'Cache', 'Backup', 'Replica'),
+  CategoryRealm: Enum('Global', 'System', 'Server'),
+  CategoryFamily: Enum('Registry', 'Dictionary', 'Table', 'Log', 'View'),
```

Comment:
Shouldn't we have `'File'` family? If not, how should we declare that something is a `File`?

Reply:
`Identifier.Category` -> 'File.schema'